src/fakedls/neo/request/GetRange.d

```diff
@@ -41,6 +41,12 @@ public class GetRangeImpl_v2: GetRangeProtocol_v2
     import ocean.text.regex.PCRE;
     import dlsproto.common.GetRange;
 
+    /// Request code / version. Required by ConnectionHandler.
+    static immutable Command command = Command(RequestCode.GetRange, 2);
+
+    /// Request name for stats tracking. Required by ConnectionHandler.
+    static immutable istring name = "GetRange";
```

Comment:
The requests derive from `IRequest`, and will always need `addHandler`, which, in turn, needs the `name` and the `command` members from the request class. However, the current definition of `IRequest` does not enforce having these members.
Do you think that it will be useful to add `getName` and `getCommand` methods to the `IRequest` interface? At this point, it will be trivial to add this change in swarm and *proto.